apps/react-app/src/common/forms/mf-button-styles.ts

```diff
@@ -0,0 +1,16 @@
+import { css } from '@emotion/css';
+import { ColorPrincipal, ColorSecondary } from './mf-color.constants';
+
+export const buttonStyle = css`
+  background-color: ${ColorPrincipal};
+  border: none;
+  padding: 15px;
+  border-radius: 8px;
+  font-family: Avenir, sans-serif;
```

Comment:
We could think about creating some global CSS styles or theming to hold settings like this

Adding @nasdan and @manudous to the loop

Reply:
Done.